NAMESPACE

```diff
@@ -2,21 +2,34 @@
 
 S3method(as.matrix,redist)
 S3method(print,redist)
+export(redist.adjacency)
 export(redist.combine)
 export(redist.combine.anneal)
 export(redist.combine.mpi)
+export(redist.compactness)
+export(redist.crsg)
 export(redist.diagplot)
+export(redist.distances)
 export(redist.enumerate)
 export(redist.findparams)
+export(redist.init.enumpart)
 export(redist.ipw)
+export(redist.map)
 export(redist.mcmc)
 export(redist.mcmc.anneal)
 export(redist.mcmc.mpi)
+export(redist.metrics)
+export(redist.parity)
+export(redist.prep.enumpart)
+export(redist.read.enumpart)
 export(redist.rsg)
+export(redist.run.enumpart)
 export(redist.samplepart)
 export(redist.segcalc)
 export(redist.smc)
 export(redist.smc_is_ci)
+import(ggplot2)
```

Comment:
Is it necessary to import all of `ggplot` and all of `sf`? Can we import just specific functions to conserve package size?

Reply:
Yeah, these can be broken down into only the used functions. They will need to be in Suggests, however, as they are used in the current vignette. Moving them will still reduce the size.